atest/robot/libdoc/invalid_usage.robot

```diff
@@ -18,9 +18,15 @@ Invalid option
     --invalid         option --invalid not recognized
 
 Invalid format
-    -f XXX BuiltIn ${OUT HTML}              Format must be 'HTML', 'XML' or 'XML:HTML', got 'XXX'.
-    --format XML:XXX BuiltIn ${OUT HTML}    Format must be 'HTML', 'XML' or 'XML:HTML', got 'XML:XXX'.
-    BuiltIn out.ext                         Format must be 'HTML', 'XML' or 'XML:HTML', got 'EXT'.
+    -f XXX BuiltIn ${OUT HTML}               Format must be 'HTML', 'XML' or 'LIBSPEC', got 'XXX'.
+    --format XML:XXX BuiltIn ${OUT HTML}     Format must be 'HTML', 'XML' or 'LIBSPEC', got 'XML:XXX'.
+    --format XML:HTML BuiltIn ${OUT HTML}    Format must be 'HTML', 'XML' or 'LIBSPEC', got 'XML:HTML'.
+    BuiltIn out.ext                          Format must be 'HTML', 'XML' or 'LIBSPEC', got 'EXT'.
+
+Invalid specdocformat
+    -s XXX BuiltIn ${OUT HTML}                Spec Doc Format must be 'RAW' or 'HTML', got 'XXX'.
```

Comment:
"Spec Doc Format must be ..." has pretty strange title case usage. One of these would be better:
- "Spec doc format must be..."
- "Spec documentation format must be ..."
- "--specdocformat must be ..."

I think I like the latest one most but then also others should be changed so that there's e.g. "--format must be ...".